### PR TITLE
Reduce the amount of exceptions thrown

### PR DESCRIPTION
**Pull request by Bond-009**
Removing a library is instant now.

### CODE REVIEW

File: MediaBrowser.LocalMetadata/Images/InternalMetadataFolderImageProvider.cs [hunk 1]
```diff
@@ -53,12 +59,18 @@ public List<LocalImageInfo> GetImages(BaseItem item, IDirectoryService directory
         {
             var path = item.GetInternalMetadataPath();
 
+            if (!Directory.Exists(path))
+            {
+                return new List<LocalImageInfo>();
```
Comment (JustAMan) [inline MediaBrowser.LocalMetadata/Images/InternalMetadataFolderImageProvider.cs]: I feel we should log this directory absense

File: MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs [hunk 2]
```diff
@@ -508,43 +507,44 @@ public string GetInputArgument(string[] inputFiles, MediaProtocol protocol)
...
             using (var processWrapper = new ProcessWrapper(process, this, _logger))
             {
+                _logger.LogDebug("Starting ffprobe with args {Args}", args);
                 StartProcess(processWrapper);
 
+                InternalMediaInfoResult result;
                 try
                 {
-                    //process.BeginErrorReadLine();
+                    result = await _jsonSerializer.DeserializeFromStreamAsync<InternalMediaInfoResult>(process.StandardOutput.BaseStream).ConfigureAwait(false);
```
Comment (JustAMan) [inline MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs]: wrap long line pls

File: MediaBrowser.Providers/MediaInfo/SubtitleResolver.cs [hunk 3]
```diff
@@ -49,9 +49,16 @@ public SubtitleResolver(ILocalizationManager localization, IFileSystem fileSyste
 
             startIndex += streams.Count;
 
+            string folder = video.GetInternalMetadataPath();
+
+            if (!Directory.Exists(folder))
+            {
+                return streams;
```
Comment (JustAMan) [inline MediaBrowser.Providers/MediaInfo/SubtitleResolver.cs]: maybe we should log dir absence here

**Comment by Bond-009** [comment]
@JustAMan I think we shouldn't log in those 2 places as we don't really expect them to be there.

**Comment by JustAMan** [comment]
@Bond-009 you still have conflicts here...

**Comment by Bond-009** [comment]
@JustAMan fixed

**Pull request merged**